core-build/gulp-core-build-serve/src/TrustCertTask.ts

```diff
@@ -13,7 +13,9 @@ import { ICertificate } from './certificates';
  *  and signed, but the user must trust it manually.
  */
 export default class TrustCertTask extends GulpTask<{}> {
-  public name: string = 'trust-cert';
```

Comment:
>public name: string = 'trust-cert'; [](start = 2, length = 35)

We override this in a couple of places... maybe we should keep name as a public string